explorer/explorerroutes.go

```diff
@@ -1155,6 +1155,7 @@ func (exp *explorerUI) ParametersPage(w http.ResponseWriter, r *http.Request) {
 	actualTicketPoolSize := int64(cp.TicketPoolSize * cp.TicketsPerBlock)
 	ecp := ExtendedChainParams{
 		Params:               cp,
+		MaximumBlockSize:     cp.MaximumBlockSizes[0],
```

Comment:
This is not necessary.  Just index it when executing the template.

Reply:
The reason I added this is because there a possibility for this value to something other than cp.MaximumBlockSizes[0], according to an earlier comment, 

> We can either take MaximumBlockSizes[0], or we can find a way to determine which is presently active.

Although I am unable to determine the block that is currently active, I am thinking the same field will pass that value when it is sorted out